vm/src/builtins/pystr.rs

```diff
@@ -1168,23 +1171,6 @@ impl IntoPyObject for &String {
     }
 }
 
-impl TryFromObject for std::ffi::CString {
-    fn try_from_object(vm: &VirtualMachine, obj: PyObjectRef) -> PyResult<Self> {
-        let s = PyStrRef::try_from_object(vm, obj)?;
-        Self::new(s.as_str().to_owned())
-            .map_err(|_| vm.new_value_error("embedded null character".to_owned()))
-    }
-}
-
-impl TryFromObject for std::ffi::OsString {
-    fn try_from_object(vm: &VirtualMachine, obj: PyObjectRef) -> PyResult<Self> {
-        use std::str::FromStr;
-
-        let s = PyStrRef::try_from_object(vm, obj)?;
-        Ok(std::ffi::OsString::from_str(s.as_str()).unwrap())
-    }
-}
-
```

Comment:
Were you able to compile this on your machine?

Reply:
Oh, I guess I didn't check with ssl enabled

Reply:
Ah, #2707 used this to convert into a `String` and that got in after the tests ran for your PR. I was wondering why the tests passed fine in the PR but not when it was merged.

Reply:
> I was wondering why the tests passed fine in the PR but not when it was merged.

Bors could’ve probably helped.